.github/workflows/credcheck.yml

```diff
@@ -0,0 +1,58 @@
+name: credentials-sanity-check
+
+on:
+  workflow_dispatch:
```

Comment:
This is to trigger it manually; ultimately, we could run it on a fixed schedule (e.g. daily) and notify the squad (?) when it fails.